### ISSUE TITLE
Trailing-slash 301 redirects not working when Comet Cache enabled

### ISSUE THREAD

**Issue by JordanMG** [wpsharks/comet-cache#931]
When Comet Cache is activated, trailing-slash 301 redirects (if configured) do not work on posts.

To replicate:

- create a fresh WordPress install on Nginx
- configure the "post name" permalink settings in WordPress
- confirm that posts without a trailing slash 301-redirect to the slashed version
(example.com/post-name -> example.com/post-name/)
- activate comet cache
- as a guest user who views the cached page, note that the redirect no longer takes place
(example.com/post-name -> example.com/post-name)

**Comment by lkraav** [comment]
What does the Comet Cache debug output at the cached page bottom say? What URL does it think it's showing?

EDIT my Apache-based trailing slash redirects have been working fine for a few years now.

**Comment by JordanMG** [comment]
The debug output shows the trailing slash in the URL.

**Comment by lkraav** [comment]
I'm thinking it's an `nginx` config issue. Comet Cache shouldn't even be getting a hit, if you're properly redirecting at webserver level.

**Comment by JordanMG** [comment]
Yeah, I never imagined Comet Cache would be dealing with this either, but it seems to be.

Nginx config:
`location / {
    try_files $uri $uri/ /index.php?$args ;
 }`

The issue can also be replicated on the official Comet Cache blog:

https://cometcache.com/blog
https://cometcache.com/blog/

https://cometcache.com/blog/comet-cache-v170808-rc-release-candidate
https://cometcache.com/blog/comet-cache-v170808-rc-release-candidate/
